recipes/15/15_interval_joins.md

```diff
@@ -0,0 +1,61 @@
+# 15 Interval Joins
+
+:bulb: This example will show how you can optimize joins between tables that are related in a temporal context.
```

Comment:
Do you want to position it purely as an optimization? It's also useful from a functional perspective: I only want to join rows that are close to each other in time, because everything else is irrelevant.

Reply:
Will make it more generic.